modules/visual_script/doc_classes/VisualScriptCustomNode.xml

```diff
@@ -131,7 +131,7 @@
 				The [code]inputs[/code] array contains the values of the input ports.
 				[code]outputs[/code] is an array whose indices should be set to the respective outputs.
 				The [code]start_mode[/code] is usually [constant START_MODE_BEGIN_SEQUENCE], unless you have used the [code]STEP_*[/code] constants.
-				[code]working_mem[/code] is an array which can be used to persist information between runs of the custom node.
+				[code]working_mem[/code] is an array which can be used to persist information between runs of the custom node. The size needs to be predefined using [method _get_working_memory_size]. Do not overwrite [code]working_mem[/code] using another [code]var[/code].
```

Comment:
> Do not overwrite [code]working_mem[/code] using another [code]var[/code].

That's a bit weird to add to the documentation. That's a very specific mention to a programming mistake which could have been done *anywhere* and is not related to `working_mem`. If you create a local variable that shadows a method parameter, then you're indeed taking the risk that accessing that identifier will maybe not access the instance you intended. I wouldn't call it out here for `working_mem` specifically, this should be part of the GDScript documentation (and might already be).

Reply:
Now the PR only ads " The size needs to be predefined using [method _get_working_memory_size]."